### DISCUSSION TITLE
Translation into Chinese of an English document.(3) 英文翻译为中文3

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#497] [MMDetection]
# Translation into Chinese of an English document.(3) 
 英文翻译为中文3 

Translation [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md))  and [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md))  into Chinese

将 [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_analysis_tools.md))  和 [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_dataset_prepare.md))  翻译为中文

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：timerring

**Claim Time**：2023年08月28日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/10848

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by timerring** [comment]
https://github.com/open-mmlab/mmdetection/pull/10848
